src/Apps/Collect2/Routes/Collections/Components/CollectionsGrid.tsx

```diff
@@ -59,6 +59,7 @@ export class CollectionsGrid extends Component<CollectionsGridProps> {
                   <Media greaterThan="xs">{index < 3 && <Separator />}</Media>
 
                   <EntityHeader
+                    initials=" "
```

Comment:
Neat! 👍 

Could we add a comment above line 61, to make it clear to future readers of this code that we are passing `initials=" "` so that the gray circle shows up? We do have this PR to help them figure that out, but this code "seems strange" - enough that I could easily see someone deleting it without investigating why it was written.

Commenting JSX is kind of tricky, if you haven't seen it before it looks like this: 

```jsx
{/* A JSX comment */}
<EntityHeader />
...
```

It'd be great if we could put it directly above line 62, but I don't think we can put it inside an element...so right before the `<EntityHeader` line would be adequate.

Reply:
Indeed. That empty space looks strange. Thanks for pointing out.  🤝